src/components/Comments/CommentsBar.tsx

```diff
@@ -0,0 +1,45 @@
+import { Close as CloseIcon } from '@mui/icons-material';
+import { Box, Drawer, Stack, Typography } from '@mui/material';
+import { IconButton } from '../IconButton';
+import { useCommentsContext } from './CommentsBarContext';
+import { CommentsThreadList } from './CommentsThreadList';
+
+interface CommentsBarProps {
+  resourceId?: string;
+}
+
+export const CommentsBar = ({ resourceId }: CommentsBarProps) => {
+  const { isCommentsBarOpen, toggleCommentsBar } = useCommentsContext();
+  const open = isCommentsBarOpen && !!resourceId;
+  const width = 350;
+
+  return (
+    <Drawer
+      variant="persistent"
+      open={open}
+      anchor="right"
+      elevation={0}
+      PaperProps={{
+        sx: { width, top: 65, height: 'calc(100vh - 65px)' },
+      }}
+      sx={[
+        !open && { display: 'none' },
+        { overflowY: 'auto', display: 'flex' },
+        open && { width: width, flexShrink: 0 },
+      ]}
+    >
+      <Stack alignItems="start" spacing={1}>
+        <IconButton size="small" onClick={() => toggleCommentsBar(false)}>
+          <CloseIcon />
+        </IconButton>
```

Comment:
There's no padding here between the drawer and the icon.

It's also kinda a shame it takes up its own row, though I understand the dilemma.
Maybe we could consider a "Hide" button to the left of the submit "Comment" button on the next pass?